finalPrep/modZeroHero.js

```diff
@@ -1,84 +1,151 @@
-
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+  if (dangerLevel > 50) {
+    console.log(badExcuse);
+  } else if (dangerLevel >= 10){
+    console.log(saveTheDay)
+  } else {
+    console.log("Meh. Hard pass.")
+  }
+}
+assessSituation(51, "Hulk smash!!!", "Hulk forgot to do the dishes")
 // Your function should include an if/else statement that meets the following criteria
 //   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
 //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
 //   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
 
 //Test Cases
 var announcement = 'Never fear, the Courageous Curly Bracket is here!';
+console.log(announcement);
 var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.';
+console.log(excuse);
 // assessSituation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
 //assessSituation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 //assessSituation(3, announcement, excuse) > should print - "Meh. Hard pass."
-
+assessSituation(99, announcement, excuse);
+assessSituation(21, announcement, excuse);
+assessSituation(3, announcement, excuse);
 // Declare a new variable - scaryMonster - assigned to an Object with the following key/values
 //   - name (string)
 //   - smell (string)
 //   - weight (integer)
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
-
+var scaryMonster = {
+  name: "Big tax man",
+  smell: "New office",
+  weight: 1547,
+  citiesDestroyed: ["Denver", "Colorado springs", "New york", "Dallas"],
+  luckyNumbers: [1,2,6,34,89],
+  address: {
+    number: 17248,
+    street: "nowhere st",
+    state: "New york",
+    zip: 11001
+  }
+}
+console.log(scaryMonster);
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100
+    this.energyLevel = 50
+  }
+  sayName() {
+    console.log(this.name);
+  }
+  maximizeEnergy() {
+    this.energyLevel += 950;
+  }
+  gainPower(number) {
+    this.powerLevel = number * 100;
```

Comment:
Great job making a dynamic function, though remember to make sure the logic matches the name- here, we are reassigning power, but I think it would make more sense if this method added to the current power.